reliability-engineering/pipelines/build-govsvc-docker.yml

```diff
@@ -95,6 +120,53 @@ jobs:
       tag_file: ruby-2.6.1/tag
       tag_as_latest: true
     get_params: {skip_download: true}
+
+- name: build-govsvc-aws-terraform
+  serial: true
+  plan:
+  - in_parallel:
+    - get: govsvc-aws-terraform-git
+      trigger: true
+    - get: terraform
+      trigger: true
+      params: {save: true}
+    - get: ubuntu
+      # trigger: true
```

Comment:
I guess having this commented out suggests that this won't trigger, but `trigger:  false` would be clearer and more explicit (if that works).

Reply:
didn't mean to leave that in there ... will remove.